docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -251,7 +251,7 @@ You can learn more about the new approach on the [dedicated doc page](https://mu
  </LocalizationProvider>
 ```
 
-## Adapters
+## Adapters internal changes
```

Comment:
Not sure it's the right title, but since we have other adapters-related BCs (on the dayjs adapter mostly) I think we need to clarify this title.